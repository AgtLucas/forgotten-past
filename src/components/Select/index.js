import React from 'react';
import styles from './styles.module.css';

const Select = ({ children, ...props }) => (
  <select className={styles.select} {...props}>
    {children}
  </select>
);

export default Select;