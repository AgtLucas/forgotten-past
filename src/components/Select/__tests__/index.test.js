import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import Select from '../';

describe('Select', () => {
  it('should render Select', () => {
    const props = {
    };

    const wrapper = shallow(<Select {...props} />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});