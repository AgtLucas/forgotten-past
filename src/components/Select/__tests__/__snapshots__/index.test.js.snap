// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Select should render Select 1`] = `
<select
  className="select"
>
  <option>
    Hi!
  </option>
</select>
`;
